Source/TaskDelegate.swift

```diff
@@ -137,7 +137,8 @@ open class TaskDelegate: NSObject {
         completionHandler(bodyStream)
     }
 
-    func urlSession(_ session: URLSession, task: URLSessionTask, didCompleteWithError error: NSError?) {
+    @objc(URLSession:task:didCompleteWithError:)
```

Comment:
Yes, unfortunately. I think it's a byproduct of the various subclasses protocol requirements not propagating the Swiftified version of these APIs up the hierarchy. I believe @kcharwood filed a bug about it when he did the initial Swift 3 port at WWDC. @kcharwood ?